script.js

```diff
@@ -1,4 +1,109 @@
+//Win-Lose-Draw Record
+var userCount = 0;
+var opponentCount = 0;
+var drawCount = 0;
+var totalRounds = userCount + opponentCount + drawCount;
+var myWinningPercentage = (userCount / totalRounds) * 100;
+
+var currentGameMode = `Waiting for Username`;
+var userName = ``;
```

Comment:
if you are not using any variables in your string, you can just declare userName with empty string '' instead of ``